lib/scripts.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+const parseDataAttributes = (scriptObj) => {
+
+    // Don't mutate the original object.
+    const script = Object.assign({}, scriptObj);
+
+    if (!script.dataAttributes) {
+        return script;
+    }
+
+    const dataAttributes = Object.keys(script.dataAttributes);
```

Comment:
Can't this be written more cleanly using `map`?

Reply:
`dataAttributes` and `parsedAttributes` are objects though, I'm not sure how we can use `map` without converting them between objects and arrays a few times.

I could just remove the `dataAttributes` variable and use `Object.keys(script.dataAttributes)` directly if that makes it easier to read?